app/controllers/application_controller.rb

```diff
@@ -61,13 +62,15 @@ def deny_access
     (User.current.logged? || request.xhr?) ? render_403 : require_login
   end
 
+  def require_ssl?
+    SETTINGS[:require_ssl]
+  end
+
   def require_ssl
```

Comment:
Isn't that the same as `force_ssl`?  What I'm getting at, is why we now have two systems for HTTP to HTTPS direction.